### PR TITLE
Game Balance

### PR DESCRIPTION
**Pull request by X4**
- Corrected tech tree, energy consumptions, descriptions and buildlding costs of all units and buildings for both GDI and NOD and build times of units and structures according to the original game.

See WiKi entry for more info: 
- [Game Balancing](https://github.com/OpenRA/OpenRA/wiki/Game-Balancing)
- [Unit Specific Balance](https://github.com/OpenRA/OpenRA/wiki/Unit-Specific-Balance)

### CODE REVIEW

**Comment by Phrohdoh** [comment]
I'm not a fan of reimplementing the original TD here.

File: mods/cnc/rules/aircraft.yaml [hunk 1]
```diff
@@ -1,13 +1,15 @@
 TRAN:
 	Inherits: ^Helicopter
 	Valued:
-		Cost: 750
+		# Cost: 750
```
Comment (Mailaender) [inline mods/cnc/rules/aircraft.yaml]: Please don't out-comment things, just replace them. We keep the full history of changes via git blame.

**Comment by penev92** [comment]
Hold on. Before making him remove all those comments we should decide if we're even going to use his changes. So far I haven't heard anyone say "Yes, we want those"...

**Comment by Mailaender** [comment]
Yes, the re-balancing of the cnc mod was intentional. https://github.com/OpenRA/OpenRA/wiki/FAQ#this-is-not-true-to-the-original sums it up. This may want to live in it's own thirdparty mod though. http://www.moddb.com/games/openra/mods

**Comment by DaveAxiom** [comment]
The Tiberium Dawn implementation isn't balanced and plays too much like Red Alert. The Engineers are near useless in OpenRA and Helicopters would make more sense to be built faster because of their high expense.

When I played daily, I found that players raced to build structures without regard to their defense. I'd love to see some attention on balancing to make TD players pay more attention to tactics.

**Comment by penev92** [comment]
>The Tiberium Dawn implementation isn't balanced

Is that "isn't balanced", or "isn't balanced like the original" ?

**Comment by DaveAxiom** [comment]
Players don't resort to using tactics -- it's a clear mistake to defend your base at the outset because there isn't balance. I haven't played the original in many years.

**Comment by Unit158** [comment]
:-1: The balancing for TD has been thoroughly tested and tweaked, and only changing a few parts will break gameplay significantly. We don't need another D2K.

If you're worried about the lack of tactics or certain balance problems, this is the wrong place to discuss them anyway. I would suggest making issues with specific problems to be addressed, or visit the forums at http://sleipnirstuff.com/ for more general discussion.

**Comment by MicroJOo** [comment]
the next release has a new feature that allows players to switch mods directly in the server-list ... so all our mods ra , td and d2k share the same server-list ... this will probably give TD and D2k a significant player-boost which they deserve  ... im sorry but changin the balance at this stage doesnt look like a good idea , specially when there is a little handful of regular players with no idea whats happening ... 

same as @Unit158  i also suggest getting in touch with the community first ... and start a discussion on sleipnirstuff ...

**Comment by Mailaender** [comment]
No consensus yet and unable to merge as this patch as it is most likely based on an outdated revision.

**Pull request closed without merging**